### ISSUE TITLE
Background Clip not applying webkit property

### ISSUE THREAD

**Issue by Ehesp** [tailwindlabs/tailwindcss#2300]
### Describe the problem:

Using tailwindcss 1.7.6, trying to use the Backgroud Clip on Webkit browsers doesn't seem to be working. I'm using NextJS & Chrome, so I'm not sure if it's specifically related:

Example from docs:

```
return (
    <div className="text-center text-5xl font-extrabold leading-none tracking-tight">
      <span className="bg-clip-text text-transparent bg-gradient-to-r from-teal-400 to-blue-500">
        Hello world
      </span>
    </div>
  );
```

Result:

![image](https://user-images.githubusercontent.com/842078/92107642-09228000-edde-11ea-98c7-49f38a519dce.png)

![image](https://user-images.githubusercontent.com/842078/92107671-10e22480-edde-11ea-9009-522ba4d90c1c.png)

Looking at the docs website, the `-webkit-background-clip` property hasn't been included.

### Link to a minimal reproduction:

https://github.com/Ehesp/tailwind-background-clip-issue

Basically a fresh Next.js project which uses postcss to pull in tailwind.

**Comment by adamwathan** [comment]
Hey @Ehesp! We don't include vendor prefixes in Tailwind because every project has different browser requirements, so instead we recommend using Autoprefixer to handle all of that:

https://tailwindcss.com/docs/browser-support#vendor-prefixes

**Comment by Ehesp** [comment]
Oh interesting, this is the first thing I've come across which hasn't worked out of the box 😁 Makes sense though! Just added and indeed it works. Thanks!

**Comment by adamwathan** [comment]
Awesome glad that helped! The usual culprit here is the `appearance` class, haha...

**Comment by toughyear** [comment]
@adamwathan  I am facing a similar issue with Background Clip not applying webkit property, I have used the following config as per the example from Vercel repo - 

```
module.exports = {
  plugins: [
    "tailwindcss",
    "postcss-flexbugs-fixes",
    [
      "postcss-preset-env",
      {
        autoprefixer: {
          flexbox: "no-2009",
        },
        stage: 3,
        features: {
          "custom-properties": false,
        },
      },
    ],
  ],
};

```

It already has autoprefixer plugin included with postcss-preset-env. I am facing the same issue as the OP. Any working example of this issue using NextJS and TailwindCSS?

**Comment by Skoua** [comment]
For the lazy ones, add this to your style.css:

```
.bg-clip-text {
    -webkit-background-clip: text;
    background-clip: text;
  }
```

**Comment by MinjaBaraka** [comment]
I use tailwind css,  and below is my code

"
<h1 class="mb-4 text-4xl lg:text-8xl font-bold uppercase text-white text-center bg-clip-text text-transparent bg-gradient-to-br from-amber-400 to-orange-300">Safi Schools</h1>
"
the bg-clip-text does not work and I dont what the solution for it.
Can anyone help me please...?

**Comment by mahtablog** [comment]
The problem is browser support
`The “background-clip: text” is supported in all main browsers with the Webkit prefix, it allows a background image to be clipped by a text element.`

Using Tailwind CSS: If you're using Tailwind CSS and want to apply this with utility classes, you can extend Tailwind’s configuration to include the necessary properties:

Extend Tailwind Configuration: Add the -webkit-background-clip: text to your Tailwind globals.css file or extend it through the extend section in your tailwind.config.ts if you need it globally

`@layer utilities {
  .bg-clip-text {
    -webkit-background-clip: text;
    background-clip: text;
  }
}`
